core/src/display_object/edit_text.rs

```diff
@@ -1657,27 +1657,33 @@ impl<'gc> TDisplayObject<'gc> for EditText<'gc> {
     }
 
     fn width(&self) -> f64 {
-        self.0.read().bounds.width().to_pixels()
+        let edit_text = self.0.read();
+        let text_width = edit_text.bounds.width();
+        text_width.to_pixels() * (edit_text.base.transform.matrix.a as f64)
```

Comment:
This needs to transform the bounds by the matrix using something like `BoundingBox::transform`. For example:

```
import flash.geom.Matrix;
var matrix:Matrix = new Matrix();
matrix.c = 1.5; // still affects the width
txt.transform.matrix = matrix;
// Will be something like 2.5 times the original width, because now it is skewed and the width should consider the entire AABB.
trace(txt.width); 
```